app/src/main/java/com/example/henzoshimada/feeltrip/Mood.java

```diff
@@ -148,7 +148,7 @@ public String getDescription() {
         return description;
     }
 
-    public void setDescription(String description) throws DescriptionTooLongException {
+    public void setDescription(String description, String append) throws DescriptionTooLongException {
```

Comment:
Again, this functionality can be changed. Personally, I don't see an issue with it, other than we store the same String twice in Elasticsearch: once under "emotionalState", and once at the end of "description"